lib/filters/global/actions.js

```diff
@@ -17,7 +17,12 @@ module.exports = {
                     body.format = 'json';
                     return restbase[req.method](req)
                     .then(function(res) {
-                        if (res.status !== 200) {
+                        if (
+                            res.status !== 200
```

Comment:
Doesn't look likely.  We currently call the PHP action API only from pagecontent.js, and it always supplies enough of a body to prevent this case.  Shall I drop the `res.body.query{,.pages}` checks?